From 3a80bb67bf06d28472e68c470e2c16369a2a7db5 Mon Sep 17 00:00:00 2001
From: Kshitiz Varshney <kshitiz.varshney@nxp.com>
Date: Fri, 6 Oct 2023 05:56:02 +0200
Subject: [PATCH] imx-v8-defconfig: OPTEE based Trusted key enablement for
 i.MX93

This commit modifies defconfig file for supporting OPTEE based trusted
key which is required for IMA/EVM for i.MX93 as CAAM is not present on i.MX93.

Signed-off-by: Kshitiz Varshney <kshitiz.varshney@nxp.com>
---
 arch/arm64/configs/imx_v8_defconfig | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/configs/imx_v8_defconfig b/arch/arm64/configs/imx_v8_defconfig
index d2945290a79b..2d171e3da1d9 100644
--- a/arch/arm64/configs/imx_v8_defconfig
+++ b/arch/arm64/configs/imx_v8_defconfig
@@ -1149,3 +1149,7 @@ CONFIG_IMA_WRITE_POLICY=y
 CONFIG_IMA_READ_POLICY=y
 
 CONFIG_IMX_EL_ENCLAVE=y
+
+# enable OPTEE based trusted key (for i.MX93)
+CONFIG_TRUSTED_KEYS_TEE=y
+CONFIG_TRUSTED_KEYS_CAAM=n
-- 
2.25.1

